crowbar_framework/app/helpers/barclamp/ceph_helper.rb

```diff
@@ -35,5 +35,17 @@ def diskmodes_for_ceph(selected)
         selected.to_s
       )
     end
+    def replicas_number_for_ceph(selected)
+      options_for_select(
+        [
+          ["Automatically","0"],
```

Comment:
s/Automatically/Automatic/

Btw, these strings should probably be marked for translations.